compiler/src/iree/compiler/Dialect/Flow/Transforms/Passes.cpp

```diff
@@ -248,7 +248,11 @@ void addDispatchRegionCreationPasses(OpPassManager &passManager,
       // Collapse dimensions of linalg Ops.
       .addPass(IREE::Flow::createCollapseDimensionsPass)
       // Convert dispatch regions into dispatch workgroups by capturing values.
-      .addPass(IREE::Flow::createFormDispatchWorkgroupsPass);
+      .addPass(IREE::Flow::createDispatchRegionsToWorkgroupsPass)
```

Comment:
It might also be good to move the passes from line 227 here (i.e. `createDispatchWithTransformDialectPass` to the last pass added here to a "pipeline" similar to `flowDispatchRegionFormationPreprocessingPipeline` below.

Reply:
Should this new pipeline be registered? Also, im not sure if `addDispatchRegionMaterializationPasses` is a good name

Reply:
Would be nice to register it. Then your lit test changes below can just invoke the pipeline?

Reply:
Yes, and that would fix the issue I was having with reordered args.